Final_project_2/frontend/css/style.css

```diff
@@ -485,26 +480,34 @@ address{
   margin-left: 80px;
   margin-bottom: 70px;
 }
+
 address .fa {
   display: block;
   font-size: 33px;
   margin-bottom: 20px;
 }
 
+.address-item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
 
-.align {
+.half-width {
```

Comment:
renamed the class to be more specific about what it does. This is good practice to avoid clashing CSS class names and to make the code more understandable.